README.md

```diff
@@ -4,3 +4,27 @@ fwbuilder
 =========
 
 Firewall Builder is a GUI firewall management application for iptables, PF, Cisco ASA/PIX/FWSM, Cisco router ACL and more. Firewall configuration data is stored in a central file that can scale to hundreds of firewalls managed from a single UI.
+
+
+Installation instructions
+=========================
+
+
+Ubuntu
+---------
+
+run the following apt-get install command to install the compile requirements:
+
+ sudo apt-get install git automake autoconf libtool libxml2-dev libxslt-dev libsnmp-dev qt4-dev-tools
```

Comment:
Please update to using `qtbase5-dev-tools` as Qt4 is end of life since December 2015.

Firewall Builder has supported Qt4 and Qt5 since August 2013 (https://github.com/fwbuilder/fwbuilder/commit/96268831a2c948ce5fa549032d39ad7a486221e0) but Qt4 support will be removed at some point.

Debian is in the process of removing Qt4: https://wiki.debian.org/Qt4Removal